### ISSUE TITLE
VAMC facility map fails to load on some detail views

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#28558]
## What happened?
I noticed some maps were failing to load on the VAMC `/locations` views. The async fetch was trying to call a VHA endpoint that didn't exist. Screenshot and steps to reproduce below.

**UPDATE August 17**
The VSP Testing Tools Team is working on porting our sitemap tests from Nightwatch to Cypress on GitHub Actions ( https://github.com/department-of-veterans-affairs/va.gov-team/issues/22443) and this has been one of the challenges. Right now we're considering ignoring the map altogether. Where the map is loaded async after the page is rendered, it has lead to tests running inconclusively with false positives and false negatives.

## Specs:
- Device: Macbook
- Browser: Chrome latest

## Steps to Reproduce
- URLs (updated August 17):
  - https://www.va.gov/atlanta-health-care/locations/east-point-va-clinic/
  - https://www.va.gov/albany-health-care/locations/troy-va-clinic/
  - https://www.va.gov/atlanta-health-care/locations/fulton-county-va-clinic/
  - https://www.va.gov/battle-creek-health-care/locations/lansing-north-va-clinic/
  - https://www.va.gov/black-hills-health-care/locations/mission-va-clinic/
  - https://www.va.gov/black-hills-health-care/locations/wessington-springs-va-clinic/
  - https://www.va.gov/eastern-colorado-health-care/locations/denver-va-domiciliary/
  - https://www.va.gov/hudson-valley-health-care/locations/montrose-2-va-mobile-clinic/
  - https://www.va.gov/northeast-ohio-health-care/locations/cleveland-2-va-mobile-clinic/
  - https://www.va.gov/northeast-ohio-health-care/locations/mccafferty-va-clinic/
  - https://www.va.gov/puget-sound-health-care/locations/bellevue-va-clinic/
  - https://www.va.gov/puget-sound-health-care/locations/federal-way-va-clinic/
  - https://www.va.gov/puget-sound-health-care/locations/north-seattle-va-clinic/#prepare-for-your-visit
  - https://www.va.gov/saginaw-health-care/locations/grand-traverse-va-clinic/
- Open Chrome dev tools and click on the Network tab
- Reload any of the URLs above
- Look for a Fetch/XHR response that starts with **VHA_NUMBER** and verify it returns a 404 code
- This fetch call is what renders the map
  - Can be verified by blocking this call on another page where the map loads
  - Can also be verified by copy/pasting the API call into a new browser tab and removing any extra letters after VHA_123 in the URL string

## Screenshot
![Screen Shot 2021-08-11 at 8 42 26 AM](https://user-images.githubusercontent.com/934879/129041503-42ab4fb0-4561-49ca-b268-7152b04339c7.png)

## Desired behavior
- Map should load correctly on all views

## Acceptance Criteria
- [ ] Map should load correctly
- [ ] No async calls should return `404` codes
- [ ] (Near future) pages should not fail Cypress a11y tests because map does not load properly

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with `Bug`**

**Comment by mmiddaugh** [comment]
apologies, @rroueche Dave and I both had PTO last week. Stan and I are planning to discuss tomorrow.

**Comment by davidconlon** [comment]
@mmiddaugh we need to add this to a discussion with CMS as well.
Mission VA Clinic = vha_568HJ
https://www.va.gov/find-locations/facility/vha_568HJ does not appear.

I did some spelunking based on my captured downloads from the Lighthouse Facilities API and vha_568HJ disappeared from LH sometime between 3/31 and 6/09 of this year. 

So two things here:
1. @SgardnerPMP - can you check to see if these facilities have been archived in VAST? If they have, can you validate with PAOs that they should be archived and then archive the relevant facility and facility service nodes? That should fix the immediate problem.
2. @mmiddaugh strategically we need to elevate the backlogged issue with CMS about being able to track and validate changes in VAST into CMS so that we can adjust as-needed

**Comment by mmiddaugh** [comment]
This issue is no longer needed
